docs/README.md

```diff
@@ -1,355 +1,287 @@
-# fluidframework-docs
+# Website
 
-This is the code and content for <https://fluidframework.com>.
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
 
-The site is generated using [Hugo](https://gohugo.io/).
+## Installation
 
-## Website quick start
-
-To quickly get started previewing the website's contents locally, open the docs folder in a terminal and install the dependencies using npm.
-
-```bash
-npm i -g pnpm
-cd docs
-pnpm install
+```shell
+$ pnpm i
 ```
 
-Then, start the server.
-
-```bash
-npm start
-```
+## Local Development
 
-Open <http://localhost:1313> to preview the site.
+There are two options for local testing.
 
-### API documentation and Playground
+### `build:generate-content` and `start`
 
-The steps above won't include API documentation (the TSDoc JSON files) by default.
+The easiest way to get started testing the website is to leverage Docusaurus's `start` functionality.
+This command starts a local development server and opens up a browser window.
+Most changes are reflected live without having to restart the server.
 
-To include generated API documentation in your local preview, you can run a complete build from the repo root.
-Then run the `build:api` script from this directory.
+Before you can use this, you'll need to ensure our API documentation has been built.
+So start by running:
 
-```bash
-npm run build:api
+```shell
+npm run build:generate-content
 ```
 
-Alternatively, you can skip the complete repo build by downloading the latest API docs and Playground files with the `download` script.
+Then, run:
 
-```bash
-npm run download
+```shell
+$ npm start
 ```
 
-> Note that the `download` script will **overwrite any locally built API docs.**
-
-## Building the documentation
-
-To build the website and run other repo-wide documentation generation, run the `build` script.
-The output will be in the `public/` folder.
+#### Limitations
 
-```bash
-npm run build
-```
+Note: the following functionality will not work in this mode.
```

Comment:
Nit. The section is already about Limitations, the "Note:" bit feels superfluous. This suggestion goes together with the next one.
```suggestion
The following functionality will not work in this mode.
```